src/components/QuickOpenModal.js

```diff
@@ -357,8 +357,20 @@ export class QuickOpenModal extends Component<Props, State> {
     return !this.getResultCount() && !!query;
   }
 
+  renderLoading = (isSymbolSearch: boolean) => {
```

Comment:
there's no reason to pass the argument `isSymbolSearch`

Reply:
I tried to use it in the function, but I guess you can't do that, or at least the way I tried. Do you mean I should put it inside the function or just use it in the render like... 
> ( this.isSymbolSearch() && this.renderLoading() )